sway-core/src/semantic_analysis/namespace/module.rs

```diff
@@ -144,6 +146,8 @@ impl Module {
 
     /// Insert a submodule into this `Module`.
     pub fn insert_submodule(&mut self, name: String, submodule: Module) {
+        let mut submodule = submodule;
```

Comment:
Code style nit: remove this LOC and add `mut` in front of the argument `submodule`.